src/map/itemdb.hpp

```diff
@@ -7,6 +7,8 @@
 #include "../common/db.hpp"
 #include "../common/mmo.hpp" // ITEM_NAME_LENGTH
 
+///Minimum available Item ID - 1
+#define ITEMID_DUMMY 499
```

Comment:
Check a few lines below. Should be part of the enum! :-)

Reply:
Thought about that, but... "for hardcoded purpose"? Seems a bit vague .)
Do you mean to add new ITEMID_CHOKO_GANGJEONG, or ITEMID_DUMMY ? Or both?

Reply:
O-ok. I've figure it out .)